docs/mindsdb-docs/docs/sql/tutorials/customer-churn.md

```diff
@@ -1,3 +1,4 @@
+## Introduction
 
 MindsDB as a Machine Learning framework can help marketing, sales, and customer retention teams determine the best incentive and the right time to make an offer to minimize customer turnover.
 
```

Comment:
I think we can remove most of the introduction for this tutorial in sake of having standardized intros between Tutorials. (maybe adding backlinks to Setups)

Reply:
Ok. I will standardize the introduction parts for all three tutorials.